css/properties/text-decoration-skip.json

```diff
@@ -40,16 +40,28 @@
               "version_removed": "46",
               "notes": "Only supports the deprecated <code>ink</code> value."
             },
-            "safari": {
-              "version_added": "8",
-              "prefix": "-webkit-",
-              "notes": "Only supports the <code>none</code> and <code>skip</code> values; all other values behave like those two values."
-            },
-            "safari_ios": {
-              "version_added": "8",
-              "prefix": "-webkit-",
-              "notes": "Only supports the <code>none</code> and <code>skip</code> values; all other values behave like those two values."
-            },
+            "safari": [
+              {
+                "version_added": "12.1",
+                "notes": "Only supports the <code>none</code> and <code>skip</code> values; all other values behave like those two values."
+              },
+              {
+                "version_added": "8",
+                "prefix": "-webkit-",
+                "notes": "Only supports the <code>none</code> and <code>skip</code> values; all other values behave like those two values."
+              }
+            ],
+            "safari_ios": [
+              {
+                "version_added": "12.2",
+                "notes": "Only supports the <code>none</code> and <code>skip</code> values; all other values behave like those two values."
```

Comment:
I'm not sure when/if this was fixed so I just kept it.

Reply:
I think that's a sensible call. Can't find WPT tests to verify if this got fixed or not either, but let's leave the note for now.